glean-core/ios/Glean/Metrics/Ping.swift

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+/// This implements the developer facing API for custom pings.
+///
+/// Instances of this class type are automatically generated by the parsers at build time.
+///
+/// The Ping API only exposes the `Ping.send` method, which schedules a ping for sending.
+public class Ping {
+    var handle: UInt64
+    let name: String
+    let includeClientId: Bool
+
+    /// The public constructor used by automatically generated metrics.
+    public init(name: String, includeClientId: Bool) {
+        self.name = name
+        self.includeClientId = includeClientId
+        self.handle = glean_new_ping_type(name, includeClientId ? 1 : 0)
```

Comment:
Didn't you add an extension for this..   something like `getByte` or `toByte`?  I thought I saw that somewhere but now I don't see it.  It seems like that might be a good extension for readability if we have to do much Bool -> UInt8 conversion.

Reply:
I ... actually might have done that. Fixing in a followup